### PR TITLE
Feature/read new access tokens

### PR DESCRIPTION
**Pull request by erichulburd**

### CODE REVIEW

File: integration_test.sh
Comment (rigettizach) [inline integration_test.sh]: What's this for?

File: integration_test.sh
Comment (rigettizach) [inline integration_test.sh:20]: Not for this PR, but in the future we may want to augment commands with a `--yes|-y` option to bypass confirmation prompts rather than using this echo technique.

File: integration_test.sh
Comment (rigettizach) [inline integration_test.sh:37]: This could be simplified a little to `$RESERVATIONS =~ [[:digit:]]+` and referencing `$BASH_REMATCH` after. The reservation id is the first sequence of digits in the output.

File: src/config.ts
Comment (rigettizach) [inline src/config.ts:157]: What's this bit for?

File: src/config.ts
Comment (rigettizach) [inline src/config.ts:192]: Should this reset to 0 when it succeeds? Or is the object ephemeral?

**Review by rigettizach — CHANGES_REQUESTED** [review]
I have a couple questions before approval, but nothing major.

File: src/config.ts
Comment (erichulburd) [reply src/config.ts:157]: This is checking forest server URL. If it is localhost:8000 - it will point the user to localhost:3000 for the UI.

File: src/config.ts
Comment (erichulburd) [reply src/config.ts:192]: Well, it should be practically impossible for the token to expire over the course of a single CLI command.

File: integration_test.sh
Comment (erichulburd) [reply integration_test.sh:20]: Done just this in a qcs-cli PR: https://github.com/rigetti/qcs-cli/pull/15 That is the intention of the `--confirm` flag.

Regardless, we'll keep mocking out responses for this user flow here.

File: integration_test.sh
Comment (erichulburd) [reply integration_test.sh]: I should have gotten rid of this - must have slipped.

File: src/config.ts
Comment (rigettizach) [reply src/config.ts:157]: Why does it do that?

Comment (erichulburd) [reply src/config.ts:157]: If there are no auth credentials in `~/.qcs` directory it tells them to go to `{qcsURL}/auth/token` to download credentials.

**Review by rigettizach — APPROVED** [review]

**Pull request merged**
